config/locales/en/course/assessment/assessments.yml

```diff
@@ -39,6 +39,7 @@ en:
             text_response: 'Text Response'
             file_upload: 'File Upload'
             programming: 'Programming'
+            scribing: 'Scribing'
```

Comment:
This will cause Travis builds for future PRs to fail until the new question link is added, I think it's better to remove it first.